registry/client.go

```diff
@@ -352,3 +352,118 @@ func (e Error) Error() string {
 
 	return "registry error: " + strconv.Itoa(e.StatusCode)
 }
+
+// Compatibility levels.
+const (
+	BackwardCL           string = "BACKWARD"
+	BackwardTransitiveCL string = "BACKWARD_TRANSITIVE"
+	ForwardCL            string = "FORWARD"
+	ForwardTransitiveCL  string = "FORWARD_TRANSITIVE"
+	FullCL               string = "FULL"
+	FullTransitiveCL     string = "FULL_TRANSITIVE"
+	NoneCL               string = "NONE"
+)
+
+/*
+this function returns whether or not a compatibility level
+is valid according to the ones described in:
+https://docs.confluent.io/platform/current/schema-registry/develop/api.html#compatibility
+.
+*/
+func validCompatibilityLevel(compatibilityLevel string) bool {
```

Comment:
It is more common for validation functions to return errors, and `compatibilityLevel` is very verbose, `lvl` is more than enough to know what it is:
```suggestion
func validateCompatibilityLevel(lvl string) error {
```

Reply:
done